gems/sorbet/lib/hidden-definition-finder.rb

```diff
@@ -105,7 +105,7 @@ def write_constants
       [
         File.realpath("#{__dir__}/../bin/srb"),
         'tc',
-        '--print=symbol-table-json',
+        '--print=symbol-table-full-json',
```

Comment:
I am not sure if we really need `symbol-table-full-json` here. It feels like this can now be changed to use the short version, thus saving on processing time, but I couldn't be sure.